packages/now-python/src/index.ts

```diff
@@ -198,3 +185,6 @@ export const build = async ({
 };
 
 export { shouldServe };
+
+// other packages can depend on @now/python to reuse common logic
+export { installRequirement, installRequirementsFile };
```

Comment:
There are no other packages that should depend on `@now/python`. You can remove this export.

Reply:
I am building a runtime that is based on python and I would like to share some logic, can we leave the export please?

Reply:
The problem is that these exports are not part of the [ZEIT Runtime API](https://github.com/zeit/now/blob/master/DEVELOPING_A_RUNTIME.md) so someone might change the way pip install works or just rename the function and it would be an unintentional breaking change for your tool that depends on this export.

If you really want to keep it, let's add a comment that says "internal only - expect breaking changes if other packages depend on these exports"

Reply:
Actually, I changed my mind. Remove the export here.

Then your package can do something like:

```js
import { installRequirement, installRequirementsFile } from '@now/python/dist/install';`
```

This will make it clear that is is not a public API.

Reply:
ncc creates a single index.js file, i cannot import from install.js

Reply:
can we leave these export and merge?

Reply:
Please update the comment to "internal only - expect breaking changes if other packages depend on these exports"